suite-native/module-onboarding/src/screens/GetStartedScreen.tsx

```diff
@@ -23,13 +24,24 @@ type NavigationProps = StackToStackCompositeNavigationProps<
 
 export const GetStartedScreen = () => {
     const navigation = useNavigation<NavigationProps>();
+    const { userColorScheme } = useUserColorScheme();
+    const systemColorScheme = useSystemColorScheme();
 
     const handleRedirect = () => {
         navigation.navigate(RootStackRoutes.AccountsImport, {
             screen: AccountsImportStackRoutes.SelectNetwork,
         });
     };
 
+    const getImageSource = () => {
```

Comment:
just a nit: It could be handled as a const above the component.

```tsx
const themeToImageMap  = {
    standard : "...path",
    dark: "...dark path"
} as const satisfies Record<ColorScheme, string>;
```

But I don't insist on this and will approve it anyway.

Reply:
It's not recommended to do dynamic require so I leave it as it it

Reply:
You could put just the path string in the map. But as I said, not big deal.

Reply:
Yes, that's still going against the recommended way of doing this though

Reply:
@PeKne check this.
https://stackoverflow.com/questions/30854232/react-native-image-require-module-using-dynamic-names